test/integration/lib/html/philips_initial.html

```diff
@@ -0,0 +1,477 @@
+Bad JSON: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```

Comment:
I included  this because the first thing foxbox does is to get '/', and this is what philips hue returns, I suspect foxbox is only concerned about '200 OK' response, but I figured maybe I'll put the whole page just to be safe.